authorizer/task_test.go

```diff
@@ -134,22 +142,18 @@ func TestValidations(t *testing.T) {
 		Bucket:                 "holder",
 		RetentionPeriodSeconds: 1,
 	})
-	if err != nil {
-		t.Fatal(err)
-	}
+	require.NoError(t, err)
```

Comment:
context string for NoError

Reply:
Added since there's a large visual skip.